docs/source/getting-started/running-idom.rst

```diff
@@ -0,0 +1,294 @@
+    app.config.REQUEST_TIMEOUT = 60
+    app.config.RESPONSE_TIMEOUT = 60
+
+
+    @component
+    def SomeView():
+        return html.form({"action": })
+
+
+    run(SomeView, server_type=PerClientStateServer, app=app)
+
+
+Add to an Existing Web Server
+-----------------------------
+
+If you're already serving an application with one of the supported web servers listed
+above, you can add an IDOM to them as a server extension. Instead of using the ``run()``
+function, you'll instantiate one of IDOM's server implementations by passing it an
+instance of your existing application:
+
+.. code-block::
+
+    from sanic import Sanic
+
+    from idom import component, html
+    from idom.server.sanic import PerClientStateServer, Config
+
+    existing_app = Sanic(__name__)
+
+
+    @component
+    def IdomView():
+        return html.h1("This is an IDOM App")
+
+
+    PerClientStateServer(IdomView, app=existing_app, config=Config(url_prefix="app"))
+
+    existing_app.run(host="127.0.0.1", port=8000)
+
+To test that everything is working, you should be able to navigate to
+``https://127.0.0.1:8000/app`` where you should see the results from ``IdomView``.
+
+
+Embed in an Existing Webpage
+----------------------------
+
+IDOM provides a Javascript client called ``idom-client-react`` that can be used to embed
+IDOM views within an existing applications. This is actually how the interactive
+examples throughout this documentation have been created. You can try this out by
+embedding one the examples from this documentation into your own webpage:
+
+.. tab-set::
+
+    .. tab-item:: HTML
+
+        .. literalinclude:: _static/embed-doc-ex.html
+            :language: html
+
+    .. tab-item:: ▶️ Result
+
+        .. raw:: html
+            :file: _static/embed-doc-ex.html
+
+.. note::
+
+    For more information on how to use the client see the :ref:`Javascript API`
+    reference. Or if you need to, your can :ref:`write your own server implementation
+    <writing your own server>`.
+
+As mentioned though, this is connecting to the server that is hosting this
+documentation. If you want to connect to a view from your own server, you'll need to
+change the URL above to one you provide. One way to do this might be to :ref:`add IDOM
+to your existing python web server`. Another would be to run IDOM in adjacent web server
+instance that you coordinate with something like `NGINX <https://www.nginx.com/>`__. For
+the sake of simplicity, we'll assume you do something similar to the following in an
+existing Python app:
+
+.. tab-set::
+
+    .. tab-item:: main.py
+
+        .. literalinclude:: _static/embed-idom-view/main.py
+            :language: python
+
+    .. tab-item:: index.html
+
+        .. literalinclude:: _static/embed-idom-view/index.html
+            :language: html
+
+After running ``python main.py``, you should be able to navigate to
+``http://127.0.0.1:8000/index.html`` and see:
+
+.. image:: _static/embed-idom-view/screenshot.png
```

Comment:
Put a border or something around this so that it doesn't look strange on the light theme.